### PR TITLE
add contributing doc

### PR DESCRIPTION
**Pull request by meandavejustice**
addresses #21

### CODE REVIEW

**Comment by pdehaan** [comment]
**PROTIP:** Changing the commit message to `Fixes #21` will automatically close the referenced issue when this PR gets merged.

**Comment by meandavejustice** [comment]
@pdehaan awesome! thank you

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Contribution Guidelines for the Idea Town
+
+Anyone is welcome to help with Idea Town. Feel free to get in touch with other community members on IRC, the
+mailing list or through issues here on GitHub.
+
+- IRC: `#idea-town` on `irc.mozilla.org`
```
Comment (pdehaan) [inline CONTRIBUTING.md]: `#ideatown` or `#idea-town`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Patches should be submitted as pull requests (PR).
+
+Before submitting a PR:
+- Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviewed. 
+  - Run `gulp lint` to make sure your code passes linting.
+  - Run `npm test` to make sure all tests still pass.
+- Your patch should include new tests that cover your changes. It is your and your reviewer's responsibility to ensure your patch includes adequate tests.
+
+When submitting a PR:
+- You agree to license your code under the project's open source license ([MPL 2.0](/LICENSE)).
```
Comment (pdehaan) [inline CONTRIBUTING.md]: Do we have a /LICENSE file in this repo? (We should, but I don't currently see one -- we should steal the one at https://raw.githubusercontent.com/mozilla/fxa-content-server/master/LICENSE)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+The footer should contain any information about **Breaking Changes** and is also the place to
+reference GitHub issues that this commit **Closes**.
+
+## Gulp Commands
+
+[Gulp](http://gulpjs.com/) is used to run common tasks to build, test, and run local servers.
+
+| Task | Description |
+|------|-------------|
+| `gulp lint` | run linters (JSLINT) on client side and testing JavaScript. |
```
Comment (pdehaan) [inline CONTRIBUTING.md]: JSLINT -> JSHint

**Comment by johngruen** [comment]
Thanks @meandavejustice

**Comment by johngruen** [comment]
@meandavejustice also, FWIW, sometimes travis fails just because and you have to kick it off again.

**Pull request merged**